index.js

```diff
@@ -1,187 +1,210 @@
-const github = require('@actions/github');
-const Base64 = require('js-base64').Base64;
-const YAML = require('yaml');
-
+const github = require("@actions/github");
+const Base64 = require("js-base64").Base64;
+const YAML = require("yaml");
+const process = require("process");
 
 // Environmment ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 const myToken = process.env.TOKEN;
 const octokit = new github.GitHub(myToken);
 
-
 // Constants ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-const AWS_ECR_URL = 'public.ecr.aws/cds-snc';
-
+const AWS_ECR_URL = "public.ecr.aws/cds-snc";
 
 // Logic ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 const getCommitMessages = async (repo, sha) => {
   const { data: commits } = await octokit.repos.listCommits({
     owner: "cds-snc",
     repo,
-    per_page: 50
+    per_page: 50,
   });
   let index = 0;
-  for(i = 0; i < 50; i++){
-    if(commits[i].sha.startsWith(sha)){
+  for (let i = 0; i < 50; i++) {
+    if (commits[i].sha.startsWith(sha)) {
       index = i;
       break;
     }
   }
-  return commits.slice(0, index).map(c => `- [${c.commit.message.split("\n\n")[0]}](${c.html_url}) by ${c.commit.author.name}`)
-}
-
-const getHeadSha = async (repo, branch = 'master') => {
+  return commits
+    .slice(0, index)
+    .map(
+      (c) =>
+        `- [${c.commit.message.split("\n\n")[0]}](${c.html_url}) by ${
+          c.commit.author.name
+        }`
```

Comment:
I was pleased with that. The code formatter actually replaced the `\n\n` characters with actual line breaks to make it more readable. The cascading style for each function is a classic formatting style for function chaining; also pleased there.